### PR TITLE
Create Add-SPOListDesign.md

### PR DESCRIPTION
**Pull request by SaladiHarini**
Adding documentation for new cmdlets associated with the Custom List Templates feature.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Add-SPOListDesign
+
+## SYNOPSIS
+Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams or Office.com.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:11]:
```suggestion
Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+## SYNTAX
+
+```
+Add-SPOListDesign -Title <String> -SiteScripts <SPOSiteScriptPipeBind[]> [-Description <String>]
+ [-ThumbnailUrl <String>] [-TemplateFeatures <String[]>] [-ListColor <SPOListDesignColor>]
+ [-ListIcon <SPOListDesignIcon>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams or Office.com.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:22]:
```suggestion
Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams or Office.com.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## EXAMPLES
+
+### Example 1
+
+This example creates a new list design 
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:29]:
```suggestion

```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+```powershell
+Add-SPOListDesign
+    -Title "Contoso customer tracking"  
+    -Description "Tracks key customer data in a list"  
+    -SiteScripts "<ID>"  
+    -ListColor Orange  
+    -ListIcon BullseyeTarget  
+    -Thumbnail "https://contoso.sharepoint.com/SiteAssets/site-thumbnail.png" }}
+```
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:39]:
```suggestion

This example creates a new list design 

```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+## PARAMETERS
+
+### -Description
+The display description of the list design.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:43]:
```suggestion
The display description of the list design.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ListColor
+Default color associated with the list, when creating the list.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:58]:
```suggestion
Default color associated with the list when creating the list.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ListIcon
+Default icon associated with the list, when creating the list.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:74]:
```suggestion
Default icon associated with the list when creating the list.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TemplateFeatures
+Set of features that the template comes with. It is displayed as a bulleted list, when user is looking at the template preview in the list creation dialog.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:105]:
```suggestion
Set of features that the template comes with. It is displayed as a bulleted list when the user is looking at the template preview in the list creation dialog.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Title
+The display name of the list design
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:135]:
```suggestion
The display name of the list design.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:150]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### System.Object
+## NOTES
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:159]:
```suggestion

## NOTES
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 12]
```diff
@@ -0,0 +1,161 @@
...
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:154]:
```suggestion
None
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 13]
```diff
@@ -0,0 +1,161 @@
...
+
+## OUTPUTS
+
+### System.Object
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:158]:
```suggestion
System.Object
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SaladiHarini, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@SaladiHarini Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @SaladiHarini, one more minor change, please.

**Comment by yogkumgit** [comment]
@SaladiHarini

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+
+### Example 1
+```powershell
+Add-SPOListDesign
+    -Title "Contoso customer tracking"  
+    -Description "Tracks key customer data in a list"  
+    -SiteScripts "<ID>"  
+    -ListColor Orange  
+    -ListIcon BullseyeTarget  
+    -Thumbnail "https://contoso.sharepoint.com/SiteAssets/site-thumbnail.png" }}
```
Comment (alinsirbu) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:34]: These curly braces should be removed

**Review by JohanFreelancer9 — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+ [-ListIcon <SPOListDesignIcon>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+
+## EXAMPLES
+
+### Example 1
+```powershell
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md:27]: please add a new line between the example 1 text and the start of the example

**Review by get-itips — CHANGES_REQUESTED** [review]
Build is currently failing, can you check or maybe Generate with PlatyPS?

**Review by get-itips — CHANGES_REQUESTED** [review]
Please add author metadata

**Comment by SaladiHarini** [comment]
@tseward @get-itips @yogkumgit I've made the necessary changes to this PR and also incorporated some best practices from feedback on adjacent PRs, please review and let me know if we are good to go here. Thanks!

**Comment by yogkumgit** [comment]
@get-itips @tseward Please review.

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks

**Comment by SaladiHarini** [comment]
@tseward I've made requested changes!

**Pull request merged**